wp-includes/components/class-wp-service-worker-navigation-routing-component.php

```diff
@@ -424,6 +424,7 @@ public function serve( WP_Service_Worker_Scripts $scripts ) {
 			'BLACKLIST_PATTERNS'               => wp_service_worker_json_encode( $this->get_blacklist_patterns() ),
 			'SHOULD_STREAM_RESPONSE'           => wp_service_worker_json_encode( $should_stream_response ),
 			'STREAM_HEADER_FRAGMENT_QUERY_VAR' => wp_service_worker_json_encode( self::STREAM_FRAGMENT_QUERY_VAR ),
+			'ERROR_MESSAGES'                   => wp_service_worker_json_encode( wp_service_worker_get_error_messages() ),
```

Comment:
@westonruter: Continued with the PR and addressed your comments, too.

Question about `wp_service_worker_get_error_messages()` -- do you think this method is necessary as a global method or we could just filter it here within Navigation Component?

Reply:
Making it a method of the navigation routing component would probably be best.